app/src/store/bootstrap.ts

```diff
@@ -1,3 +1,4 @@
+import AsyncStorage from '@react-native-async-storage/async-storage';
```

Comment:
Why use this instead of just using the redux store?

Reply:
`AsyncStorage` persists through JS runtimes, whereas the Redux store is wiped and reset every time the app is launched.